index.js

```diff
@@ -388,6 +393,20 @@ module.exports = class NoiseSecretStream extends Duplex {
     this._rawStream.write(buf)
   }
 
+  _setupSecretBox (handshakeHash) {
+    this._boxSecret = b4a.allocUnsafeSlow(64) // encrypt<32>, decrypt<32>
+    this._boxSeq = b4a.allocUnsafeSlow(16) // increment<8>, initial<8>
```

Comment:
alloc both in one go, and just store the subarrays

Reply:
gotcha